src/style.css

```
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  display: block;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 .stars_star {
+  background-image: url(images/star.svg);
+}
+
+.stars--5 .stars_star {
+  background-image: url(images/star-active.svg);
+}
```

Accodring to checklist:
 _If several selectors MUST always have the same styles, group them using , to prevent accidental out ot sync in future_
```suggestion
.stars--1 :nth-child(-n+1),
add all needed selectors after comma
{
  background-image: url(images/star-active.svg);
}
```